e2e/e2e_test.sh

```
@@ -1023,16 +1023,76 @@ test_oc_permissions_injection() {
+        log_fail "JSONC root-key assertion accepted malformed JSONC"
+    else
+        log_pass "JSONC root-key assertion rejects malformed JSONC"
+    fi
+
+    if (
+        command() {
+            if [ "$1" = "-v" ] && [ "$2" = "node" ]; then
+                return 1
+            fi
+            builtin command "$@"
+        }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)